examples/pump-app/pump-common/gen/CHIPClientCallbacks.cpp

```diff
@@ -129,6 +129,29 @@ constexpr uint16_t kByteSpanSizeLengthInBytes = 2;
         return true;                                                                                                               \
     }
 
+#define GET_ATTRIBUTE_RESPONSE_CALLBACKS(name)                                                                                     \
```

Comment:
Why is this a macro? It seems 'name' is only used for logging and you have a return true/false.

A method may be useful, however then you we would have to pass in the many arguments used here (sourceNodeId, seq, writeClient, name) or you could just generate it.

A complex macro that relies on variables defined where it is used hurts readability (not easily extensible). In this case  it may not even be obvious that the underlying code using this macro may not declare a variable  'err' (which is very common) because the macro already defines one.

Reply:
Moved to the function body, also added a TODO to move all IM non generated function to seperate file.